### DISCUSSION TITLE
Can't move draft mail to other folder ?

### DISCUSSION THREAD

**Discussion by Patatra40** [tutao/tutanota#4551] [Bugs]
Hi, is it normal that it's not possible anymore to move a draft email into another folder ? It used to be possible before.

**Comment by charlag** [comment]
Hi, yes it got restricted since the last update

**Reply by Patatra40** [reply]
Well, it's too bad. Why adding a restriction like that...

**Reply by charlag** [reply]
It might get confusing when drafts are mixed in with already sent emails. Do you have a use-case where you would need to move drafts elsewhere?

**Reply by Patatra40** [reply]
I personally write many many drafts that I finally put in different folders. I share the mailbox with other people. If the mail is finally sent it could just disappear from the draft ? That's the way hotmail works I believe ?

**Reply by franz20122** [reply]
I used moving drafts to other folders as well, e.g. when I have comments that I want to note down and keep (without sending). The restriction breaks my own policy messing up Drafts: since the change the Drafts folder is cluttered with a number of messages. Its getting impossible to manage.
Draft movable to other folders would become even more useful with "conversation" grouping.
If at least the Drafts folder could have sub-folders of its own (which is not the case at present)!
